packages/contracts/hardhat.config.ts

```diff
@@ -53,10 +60,15 @@ export const networks: {[index: string]: NetworkUserConfig} = {
     chainId: 80001,
     url: `${apiUrls.polygonMumbai}${process.env.INFURA_API_KEY}`,
   },
+  base: {
+    chainId: 8453,
+    url: `${apiUrls.base}`,
+    gasPrice: ethers.utils.parseUnits('0.001', 'gwei').toNumber(),
+  },
   baseGoerli: {
     chainId: 84531,
     url: `${apiUrls.baseGoerli}`,
-    gasPrice: 20000000000,
+    gasPrice: ethers.utils.parseUnits('0.0000001', 'gwei').toNumber(),
```

Comment:
Why are fees not working by default? I tried with a test deployment on baseGoerli. Any idea @mathewmeconry?

Reply:
I am not fully sure but I believe it is because of our old ethers version. Had the same issue with Polygon which could have been solved by defining a gas station (web2 API that provides gas information about the network in a json format)

Reply:
Ok, we then we should move to the latest ethers now.